src/Data/Morpheus/Resolve/Resolve.hs

```diff
@@ -55,9 +59,13 @@ type RootResCon m event query mutation subscription
      , EncodeCon (PublishStream m event) mutation
      , EncodeSubCon m event subscription)
 
-byteStringIO :: Monad m => (GQLRequest -> m GQLResponse) -> ByteString -> m ByteString
+eitherDecodeNoDup :: L.ByteString -> Either String GQLRequest
+eitherDecodeNoDup bs =
+  encode <$> parseOnly jsonNoDup (S.pack . UTF8.encode $ L.unpack bs) >>= eitherDecode
```

Comment:
Not sure if what I'm doing here is correct. I get a `Data.Aeson.Value` from `parseOnly` which I need to encode to `ByteString` and then decode again into `GQLRequest`. There's probably an easier way to do this?

Reply:
may we could use use `fromJSON :: FromJSON a => Value -> Result a`